_search-plugins/sql/sql/nested.md

```diff
@@ -0,0 +1,216 @@
+---
+layout: default
+title: Nested Function
+parent: SQL
+grand_parent: SQL and PPL
+nav_order: 13
+has_toc: true
+redirect_from:
+- /search-plugins/sql/nested/
+---
+
+# Nested
```

Comment:
We need to include an intro sentence to let reader know what this topic is covering. We also need to avoid stacked headings. Here's a suggestion: 

Nested functions, also known as inner functions, are functions defined within other functions. They offer advantages like improved modularity and localized functionality, and are useful for tasks like complex calculations or creating event handlers.